web/src/compositions/useVersion.ts

```diff
@@ -56,27 +59,31 @@ export function useVersion() {
       current,
       currentShort: current,
       needsUpdate: false,
+      usesNext,
     };
     return version;
   }
 
   onMounted(async () => {
     const versionInfo = await fetchVersion();
 
-    let needsUpdate = false;
+    let latest;
     if (versionInfo) {
       if (usesNext) {
-        needsUpdate = versionInfo.next !== current;
+        latest = versionInfo.next;
+      } else if (current.includes('rc')) {
```

Comment:
I am checking using '-' or using the semver package most of the time as you could also release '1.2.3-beta.0'.

Maybe using the semver package is in general a nice idea as we could check that the upstream version is newer.

Reply:
So you would ignore the check / case that the user is running a newer version than the upstream config is "aware" of for now?

Reply:
fine